security-framework-sys/src/base.rs

```diff
@@ -32,6 +32,8 @@ pub type SecCertificateRef = *mut OpaqueSecCertificateRef;
 pub enum OpaqueSecAccessRef {}
 pub type SecAccessRef = *mut OpaqueSecAccessRef;
 
+pub type SecAccessControlRef = CFStringRef;
```

Comment:
in Apple's docs this type is not a string. Note that in Rust `type` is just a weak alias, like C's typedef, and offers no type safety.

Reply:
but then the later object expects it to be a string in the attribute map, that's where I got confused (i initially used an opaque), the Apple documentation is not super explicit about how it gets converted

Reply:
ok found my way